frontend/app/views/spree/checkout/_confirm.html.erb

```diff
@@ -104,7 +104,7 @@
       </dl>
     </div>
     <div id="checkout-summary" class="w-100 order-2" data-hook="checkout_summary_box">
-      <%= render partial: 'spree/checkout/summary', locals: { order: @order } %>
+      <%= render partial: 'spree/checkout/summary', locals: { order: @order, classes: '' } %>
```

Comment:
Maybe we could use local_assigns.fetch in the rendering partial with the default value, so we don't need to copy those empty classes.
Something like:
```local_assigns.fetch :classes, default_value```

default_value will be an empty string in our case.